scripts/extensions/sams/src/components/assets/assetPreviewPanel.tsx

```diff
@@ -182,6 +183,12 @@ export class AssetPreviewPanelComponent extends React.PureComponent<IProps> {
                                 <FormLabel text={gettext('Set')} style="light" />
                                 <Text>{this.props.setName}</Text>
                             </FormRow>
+                            <FormRow>
+                                <FormLabel text={gettext('Tags')} style="light" />
+                                {this.props.asset.tags?.map((tag) => (
+                                    <Tag key={this.props.asset?.tags.indexOf(tag)} text={tag.name} onClick={noop} />
```

Comment:
Does the `Tag` UI component have a way to control if the `x` icon can not be shown? As removing a Tag in Preview is not supported

Reply:
I don't think we can remove that currently using any option available.